Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -1839,6 +1839,8 @@ eval (MenuButtonActionController (MenuButtonController.OnClick config)) state =
       _ <- animateCamera config.lat config.lng 25 "NO_ZOOM"
       pure NoAction
     ]
+eval (ChooseYourRideAction (ChooseYourRideController.ChooseVehicleAC ChooseVehicleController.NoAction)) state =
```

Comment:
@S-PRAVINKUMAR why is this change required?

Reply:
@MercyQueen I asked to add this change this is basically to just modify the state when `NoAction` is called after screen render so that choose your ride component will render with the calculated height of card, otherwise its not updating even if we do continue state and the cards/variants are not visible due to zero height